src/transformStateWithClones.js

```diff
@@ -8,6 +8,31 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const copy = { ...state };
+  const arr = [];
+
+  for (const key of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
will be better =)

Reply:
![image](https://user-images.githubusercontent.com/105542865/189685305-a865776f-b0d1-4af2-a574-f2b06e128f2f.png)